src/pricecypher/storage/file_storage.py

```diff
@@ -139,31 +140,37 @@ def from_handler_settings(cls: Type[FileStorage], settings: HandlerSettings) ->
             remote_storage_settings=settings.remote_storage_settings,
         )
 
-    def read_string(self, path: str) -> str:
-        """Read the contents of a file at the given path as string.
+    def read_df(self, path: str) -> pd.DataFrame:
```

Comment:
Put the read methods together. Same for write methods. So looks like a big change, but only added the `write_models` and `read_models` methods.